hack/dev-version/dev-version_test.go

```diff
@@ -0,0 +1,34 @@
+package main
+
+import "testing"
+
+func TestPluginToDevVersion(t *testing.T) {
+	tests := []struct {
+		pluginVersion string
+		want          string
+	}{
+		{
+			pluginVersion: "v9.0",
+			want:          "v9",
+		},
+		{
+			pluginVersion: "v9.2",
+			want:          "v92",
+		},
+		{
+			pluginVersion: "v11.0",
+			want:          "v11",
+		},
+		{
+			pluginVersion: "v12.4",
+			want:          "v124",
+		},
```

Comment:
you need test cases for e.g. `v1.24 != v12.4`

Reply:
i am not sure what you mean here as the directory never has a "." in it. we are probably safe from the following collision ("v11.0" -> "v11" and "v1.1" -> v11) as we only keep 2-3 versions in tree.